doc/command_ref.rst

```diff
@@ -926,6 +926,13 @@ Module subcommands take :ref:`\<module-spec>\ <specifying_modules-label>`... arg
     required by other installed module profiles or by other user-installed packages.
     In case no profile was provided, all installed profiles get removed.
 
+``dnf [options] module remove --all <module:stream>...``
+    Remove installed module profiles, including packages that were installed with the
+    :ref:`dnf module install <module_install_command-label>` command. Additionaly ``--all`` option
+    remove all packages based by name filtering provided by selected modules. Will not remove
+    packages required by other installed module profiles or packages provided by different modules
+    (based on package names). In case no profile was provided, all installed profiles get removed.
```

Comment:
This does not sound right to me. How about

With --all option additionally remove all packages whose names are provided by specified modules. Packages required by other installed module profiles and packages whose names are also provided by any other module are not removed.